packages/rest/src/__tests__/acceptance/coercion/coercion.acceptance.ts

```diff
@@ -1,8 +1,8 @@
+import {SchemaObject} from '@loopback/openapi-v3';
 // Copyright IBM Corp. 2019,2020. All Rights Reserved.
```

Comment:
Please move the new `import` statement after the license header. Did you perhaps forgot to run "organize imports" on save?

Reply:
vscode foo.